schematic/store/synapse.py

```diff
@@ -2295,6 +2298,9 @@ async def add_annotations_to_entities_files(
                 row["entityId"] = manifest_synapse_table_id
                 manifest.loc[idx, "entityId"] = manifest_synapse_table_id
                 entityId = ""
+                # If the row is the manifest table, do not add annotations
+            elif row["entityId"] == manifest_synapse_table_id:
```

Comment:
If `entityId` is not present by the time the code get here then it's not a problem. Because of the check below annotations won't be added if the `entityId` is not supplied.